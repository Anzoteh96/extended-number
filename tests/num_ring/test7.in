-109252129619270558648
716207989325493645459
-630554304651157947301
