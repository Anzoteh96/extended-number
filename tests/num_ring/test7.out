Enter the number base b:
Enter two numbers n and m:
b is 109252129619270558648
n mod b is 60695211609870293571
m mod b is 24958473064465404587
n + m is 85653684674335698158
n - m is 35736738545404888984
n * m is 78765520103026796625
n ** m is 100965435798443321579
